website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md

```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
 ## Overview
 
 Connecting your GitLab account to dbt Cloud unlocks exciting and compelling functionality in dbt Cloud. Once your GitLab account is connected, you can:
-- [Trigger CI builds](cloud-enabling-continuous-integration-with-github) when Merge Requests are opened in GitLab.
-- Import new repos with one click
-- Carry GitLab user permissions through to dbt Cloud IDE's git actions.
-## For Developer and Team tiers
+- Log into dbt Cloud via OAuth through GitLab.
```

Comment:
is this only for Enterprise tier?

Reply:
I need to delete this line actually. You can't "log into dbt Cloud via OAuth through GitLab.", we only support that via Github it seems